artiq/coredevice/phaser.py

```diff
@@ -760,6 +824,10 @@ def set_duc_phase(self, phase):
     @kernel
     def set_nco_frequency_mu(self, ftw):
         """Set the NCO frequency.
+        The frequency is only applied after DAC synchronisation.
+
+        Use of the NCO requires the DAC mixer to be enabled. The mixer can be
```

Comment:
And presumably also the NCO needs to be enabled? Copy paste error?

Reply:
Absolutely, The main reason for adding this was the confusion in #1630. Happy to also reference the need to enable the NCO.